src/pudl/constants.py

```diff
@@ -2476,15 +2497,27 @@
 
 column_dtypes = {
     "ferc1": {  # Obviously this is not yet a complete list...
-        "construction_year": "Int64",  # Nullable Integer
-        "installation_year": "Int64",  # Nullable Integer
-        'utility_id_ferc1': 'Int64',
-        'plant_id_pudl': 'Int64',
-        'plant_id_ferc1': 'Int64',
-        'utility_id_pudl': 'Int64',
-        'report_year': 'Int64',
+        "construction_year": pd.Int64Dtype(),
+        "installation_year": pd.Int64Dtype(),
+        'utility_id_ferc1': pd.Int64Dtype(),
+        'plant_id_pudl': pd.Int64Dtype(),
+        'plant_id_ferc1': pd.Int64Dtype(),
+        'utility_id_pudl': pd.Int64Dtype(),
+        'report_year': pd.Int64Dtype(),
         'report_date': 'datetime64[ns]',
     },
+    "ferc714": {  # INCOMPLETE
+        "report_year": pd.Int64Dtype(),
+        "utility_id_ferc714": pd.Int64Dtype(),
+        "utility_id_eia": pd.Int64Dtype(),
+        "utility_name_ferc714": pd.StringDtype(),
+        "timezone": pd.CategoricalDtype(categories=[
```

Comment:
The timezones are these long strings not the codes? I am not very familiar with timezones so I'm not insinuating that this is the wrong way to go, but this is surprising to me.

Reply:
Yeah, these are the official canonical timezone codes. They all have the form `Region/City`. Here's [the Wikipedia article](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones) about them. It's kind of a nightmare, because they really do refer to geographic areas, which each have their own particular historical behaviors that need to be tracked (e.g. dates on which standard/daylight savings time shifts have happened -- which change occasionally through legislation).  Simply specifying a UTC offset isn't enough to describe how to transform times. But storing timezone-naive UTC + a timezone code seems like an accepted best practice for storing all of the required information.